README.md

```diff
@@ -27,15 +27,15 @@ This table represents the supported components of AWS OTel Collector in 2020. Th
 | prometheusreceiver              | attributesprocessor           | `awsxrayexporter`                  | healthcheckextension   |
 | otlpreceiver                    | resourceprocessor             | `awsemfexporter`                   | pprofextension         |
 | `awsecscontainermetricsreceiver`| batchprocessor                | `awsprometheusremotewriteexporter` | zpagesextension        |
-| `awsxrayreceiver`               | memorylimiter                 | loggingexporter                    | `ecsobserver`          |
-| `statsdreceiver`                | tailsamplingprocessor         | otlpexporter                       | `awsproxy`             |
```

Comment:
tailsamplingprocessor has been removed?

Reply:
`tailsamplingprocessor` was in this table, but never actually part of the defaultcomponents and has not been a dependency in any of our releases. I checked the go.sum and package for each of the tags.